src/components/TodoApp/TodoApp.jsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState, useEffect } from 'react';
+import { TodoList } from '../TodoList';
+import { TodosFilter } from '../TodosFilter';
+
+export const TodoApp = () => {
+  const [todos, setTodos] = useState([]);
```

Comment:
```suggestion
  const [todos, setTodos] = useState(null);
```

Reply:
при таком изменении получаю ошибку

Reply:
what kind of error you got?

Reply:
first I get error - TypeError: Cannot read property 'length' of null. It's because of conditional rendering, but if I change this condition, the footer becomes visible all the time

Reply:
then just check if todos true or false